image/golem_blender_app/golem_blender_app/verifier_tools/verifier.py

```diff
@@ -222,9 +224,15 @@ def get_crop_path(parent: str, filename: str) -> str:
     print("results:")
     pprint(results)
 
-    return make_verdict(
+    loop = asyncio.get_event_loop()
+    executor = ThreadPoolExecutor()
+    make_verdict_fn = partial(
+        make_verdict,
         subtask_file_paths,
         crops,
         results,
         mounted_paths['OUTPUT_DIR'],
     )
+    verdict = await loop.run_in_executor(executor, make_verdict_fn)
```

Comment:
Could you use `golem_task_api.threading.Executor` instead? This way the thread could be cancelled like in https://github.com/golemfactory/task-api/compare/cancel_executor_tasks (concept)

Reply:
Good one!

fixed :)